db/post_migrate/20240609232736_drop_commands_from_ai_personas.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+class DropCommandsFromAiPersonas < ActiveRecord::Migration[7.0]
```

Comment:
I think you need to add the `DROPPED_COLUMNS` const here too https://github.com/discourse/discourse/blob/10b9a32abb9967b3c0876a43ff8814c7bae8f701/db/post_migrate/20240202052058_drop_badge_image_column.rb ?

Reply:
not sure why, that seems odd to force that... column dropper may though be a bit of a shortcut I could use.

Reply:
The const is apparently for the backup restore, it's in our lore topic. Not sure tho :shrug: